libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -433,7 +433,7 @@ public String getNewAuthToken() {
                         if (Looper.myLooper() == null) {
                             Looper.prepare();
                         }
-                        boolean showLoginPage = accounts.length > 1;
+                        boolean showLoginPage = accounts.length == 1;
```

Comment:
This is the fix.  `==` is correct because at least one account has to exist to make it into this code block.